pgml-extension/src/bindings/transformers/mod.rs

```diff
@@ -1,15 +1,22 @@
+use std::collections::{BTreeMap, HashMap};
 use std::io::Write;
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
 use std::str::FromStr;
-use std::{collections::HashMap, path::Path};
 
 use anyhow::{anyhow, bail, Context, Result};
+use lazy_static::*;
 use pgrx::*;
+use pyo3::ffi;
 use pyo3::prelude::*;
 use pyo3::types::PyTuple;
 use serde_json::Value;
+use std::cmp::Ordering;
+use std::env;
+use std::sync::RwLock;
```

Comment:
Use `parking_lot`, it doesn't poison the lock on panic.

Reply:
Fixed